cancer_ai/rewards.py

```diff
@@ -0,0 +1,56 @@
+   
+class Rewarder():
+    def __init__(self, config=None):
+        # competition_id to (hotkey_uid, days_as_leader)
+        self.competitions_leaders = {
```

Comment:
comment which is which, I assume value is number of days as a leader, maybe do 
```python
self.competition_leaders = [
    "melanoma-1": { 
        "hotkey": str,
        "leader_since" : datetime
     }
]
```
more visible, and use datetime of time since he became a leader, in this way we can calculate more precisely how long he's a leader

Reply:
also this structure should be serialized, because it's important to keep track who became leader when.

and this data should be in serialized in save/load_state